### ISSUE TITLE
Delete/Status Mutation 

### ISSUE THREAD

**Issue by chrisrowe** [markhuot/craftql#101]
I'd like to be able to delete entries via GraphQL but I see the schema doesn't support it, are there plans to add this functionality, or perhaps support mutating the status. Either would be good options as currently I'm changing the `authorId` to a user called `Deleted` and excluding this in my getters as a workaround

**Comment by chrisrowe** [comment]
I've crudely modified `FieldBehaviors/EntryMutationArguments.php` be adding `addBooleanArgument('enabled')` I'd forgotten `status` was a dynamic property.

I'm not confident in modifying the schema or outside this file to issue a PR but I can at least see it's possible.

**Comment by markhuot** [comment]
`dev-master` adds the `enabled` mutation argument on the `upsertEntryType` field. E.g.,

```graphql
mutation {
  upsertEntryType(id:123 enabled:false) {
    id
  }
}
```

☝️ that would disable entry `123`.

----

FYI, I had to take it a step further, because once you disable an entry you can no longer upsert it, because the default element query only looks to `live` elements. So, you can now pass a `query` argument to the `upsertEntryType` field to update disabled elements, such as,

```graphql
mutation {
  upsertEntryType(query:{id:123 status:null} enabled:true) {
    id
  }
}
```

The `query:` argument takes the same things as `craft.entries`.

**Comment by chrisrowe** [comment]
Wow, fantastic thanks @markhuot 
Do you have a timeframe for publishing the next release?

**Comment by markhuot** [comment]
Probably next week or the week after, depending on things. Are you able to run `dev-master` to confirm the fix, first?

**Comment by chrisrowe** [comment]
Yes, working great here. Thanks again.

**Comment by markhuot** [comment]
👏 excellent. I'll leave this ticket open until the next stable release is cut.

**Comment by markhuot** [comment]
[1.1.0](https://github.com/markhuot/craftql/blob/master/CHANGELOG.md#110---2018-06-27) was released on 2018-06-27 and includes this fix.

**Comment by gertst** [comment]
Hi,

I want to enable a disabled entry.

However, the suggested solution in this thread does not work: 
The mutation **upsertEntryType** seems not available in the latest (v1.3.4) CraftQL version?
I'm getting the error `"Cannot query field "upsertEntryType" on type "Mutation"` when trying to run
```
mutation {
  upsertEntryType(query:{id:1769 status:null} enabled:true) {
    id
  }
}
```

Like said above in an earlier post, the below also does not work:
```
mutation contractoradministrator_EnableDisableEntry {
  upsertPerson(id: 1769, enabled: false) {
    id
    status
  }
}
```
The error is `Could not find an entry with id 1769`, as you said, because that entry is disabled.

Any other ideas on how to enable en entry?

Thanks!
Gert
